src/backend/core/ai/thread_summarizer.py

```diff
@@ -0,0 +1,29 @@
+from core.ai.utils import get_messages_from_thread
+from core.models import Thread
+from core.services.ai_service import AIService
+
+
+def summarize_thread(thread: Thread, language: str = "fr") -> str:
+    """
+    Summarizes a thread using the OpenAI client.
+    Args:
+        thread (Thread): Thread to summarize.
+    Returns:
+        str: Summary of the thread.
+    """
+
+    # Extract messages from the thread
+    messages = get_messages_from_thread(thread)
+
+    # Prepare the prompt for the AI model
+    conversation_text = "\n\n".join([str(message) for message in messages])
+    prompt_query = "Tu es un assistant intelligent qui résume des boucles de mails. Tu dois résumer le contenu de la conversation en une ou deux lignes maximum sans préciser 'Résumé:'. Si des liens importants apparaissent dans les emails, tu dois les mentionner dans le résumé de façon claire en Markdown et intégré au résumé et sinon ne pas les mentionner."
```

Comment:
We need to store these prompts outside of the code (in a template directory?) and to translate them